app/database/group_item_store_compute_all_access.go

```diff
@@ -102,11 +86,8 @@ func (s *GroupItemStore) computeAllAccess() {
 	mustNotBeError(err)
 	defer func() { mustNotBeError(stmtMarkExistingChildren.Close()) }()
 
-	// marking 'children' groups_items_propagate as 'done'
-	const queryMarkFinishedItems = `
-		UPDATE groups_items_propagate
-		SET propagate_access = 'done'
-		WHERE propagate_access = 'children'`
+	// deleting 'children' groups_items_propagate
+	const queryMarkFinishedItems = `DELETE FROM groups_items_propagate WHERE propagate_access = 'children'`
```

Comment:
Probably you should rename `stmtMarkFinishedItems` -> `stmtRemoveProcessedItems` 
and `stmtMarkDoNotPropagate` -> `stmtDeleteDoNotPropagate`

Reply:
Fixed